process/pfcoil.py

```diff
@@ -2889,26 +2889,27 @@ def superconpf(
             or (isumat == 8)
             or (isumat == 9)
         ):  # Find temperature at which current density margin = 0
-            # Temperature range ("bracketing interval") (K)
-            a = 4
-            b = tc0m  # tc0m = critical temperature (K) for superconductor at zero field and strain
 
             if isumat == 3:
-                args = (isumat, jsc, bmax, strain, bc20m, tc0m, c0)
+                arguments = (isumat, jsc, bmax, strain, bc20m, tc0m, c0)
             else:
-                args = (isumat, jsc, bmax, strain, bc20m, tc0m)
+                arguments = (isumat, jsc, bmax, strain, bc20m, tc0m)
 
-            t_zero_margin, root_result = optimize.brentq(
+            another_estimate = 2 * thelium
+            t_zero_margin, root_result = optimize.newton(
```

Comment:
As the solver has been changed from newton-raphson to brentq to newton, can this churn be fixed up please?

Reply:
Will do using rebase